tools-local/unix/acquire-tool.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+
+# Downloads the specified tool.
+# Download URL is read from the .toolversions file.
+
+if [ -z "$1" ]; then
+    echo "Argument passed as tool name is empty. Please provide a non-empty string."
+    exit 1
+fi
+
+toolName="$1"
+lowercaseToolName="$(echo "$toolName" | awk '{print tolower($0)}')"
+strictToolVersionMatch=0
+
+if [ ! -z "$2" ]; then
+    strictToolVersionMatch="$2"
+fi
+
+
+# Checks if there is an overridden acquire-tool script.
+# If yes then, use that script to acquire the tool.
+overridden_acquire_tool()
```

Comment:
+1


---
In reply to: [107792862](https://github.com/ravimeda/corefx/pull/2#discussion_r107792862) [](ancestors = 107792862)